### PR TITLE
bump TiDB to v2.1.0

### PR DESCRIPTION
**Pull request by tennix**
Closes #212

### CODE REVIEW

File: charts/tidb-cluster/templates/config/_privileged-tidb-config.tpl [hunk 1]
```diff
@@ -228,10 +231,15 @@ grpc-keepalive-timeout = 3
...
+# Enable local latches for transactions. Enable it when
+# there are lots of conflicts between transactions.
+enabled = false
+capacity = 10240000
 
-# Socket file to write binlog.
-binlog-socket = ""
+[binlog]
+# enable to write binlog.
+enable = false
```
Comment (xiaojingchen) [inline charts/tidb-cluster/templates/config/_privileged-tidb-config.tpl]: should this field be managed in Values.yaml

File: charts/tidb-cluster/templates/config/_pump-config.tpl [hunk 2]
```diff
@@ -26,3 +26,17 @@ pd-urls = "http://{{ .Values.clusterName }}-pd:2379"
 # ssl-cert = "/path/to/drainer.pem"
 # Path of file that contains X509 key in PEM format for connection with cluster components.
 # ssl-key = "/path/to/drainer-key.pem"
+#
+#
+# we suggest using the default config of the embedded LSM DB now, do not change it useless you know what you are doing
```
Comment (gregwebs) [inline charts/tidb-cluster/templates/config/_pump-config.tpl]: useless -> unless

File: charts/tidb-cluster/templates/config/_privileged-tidb-config.tpl (hunk 1 above)
Comment (tennix) [reply charts/tidb-cluster/templates/config/_privileged-tidb-config.tpl]: The command line arguments override the configuration file.

File: charts/tidb-cluster/templates/config/_pump-config.tpl (hunk 2 above)
Comment (tennix) [reply charts/tidb-cluster/templates/config/_pump-config.tpl]: Nice catch! This is copied from tidb-binlog repo, you can also send a PR there for the typo :-)

**Pull request merged**